### ISSUE TITLE
Add 'upload config file' to home page 

### ISSUE THREAD

**Issue by ststimac** [webhintio/webhint.io#31]
Missing the 'upload config file' and 'manual options' links under the search bar.

**Comment by alrra** [comment]
> Are you planning to define it

@laukstein Sorry, I don't quite understand your question. Can you be more specific? What are you referring to by "it"?

If you are referring to the configurations for the online scanner, we will most probably allow the user to configure things manually, one by one, or upload a JSON file containing the configurations.

If you are referring to were the CLI configurations can reside, they can be in a [`.sonarwhalrc` file in the root of the project, or under `sonarwhalConfig` in the `package.json` file](https://github.com/sonarwhal/sonarwhal/blob/015d53e4b0a1a384e4929ebdbbce5d85a9fa6af4/src/lib/config.ts#L33-L38).

**Comment by laukstein** [comment]
@alrra I meant where and how would look the config file when scanning the online scanner. I think it would benefit if supported configuration located in the scanned domain.

Does [config.ts#L33-L38](https://github.com/sonarwhal/sonarwhal/blob/015d53e4b0a1a384e4929ebdbbce5d85a9fa6af4/src/lib/config.ts#L33-L38) applies also to online scanner?

**Comment by alrra** [comment]
> Does [config.ts#L33-L38](https://github.com/sonarwhal/sonarwhal/blob/015d53e4b0a1a384e4929ebdbbce5d85a9fa6af4/src/lib/config.ts#L33-L38) applies also to online scanner?

No. As [previously stated](https://github.com/sonarwhal/sonarwhal/issues/666#issuecomment-345844980), the online scanner does not yet support configurations.

> I meant where and how would look the config file when scanning the online scanner.

_"If you are referring to the configurations for the online scanner, we will most probably allow the user to configure things manually, one by one, or upload a JSON file containing the configurations."_
